### DISCUSSION TITLE
Author URLs do not appear in sidebar, despite being generated properly.

### DISCUSSION THREAD

**Discussion by masasin** [mmistakes/minimal-mistakes#3030] [Q&A] [answered]
## Environment

- Minimal Mistakes version: 4.22.0
- Ruby gem or remote theme version: Not sure
- Jekyll version: Not sure
- Git repository URL: github.com/masasin/masasin.github.io
- GitHub Pages hosted (if yes provide URL to site): masasin.github.io or jnassar.com
- Operating system: Linux

## Expected behavior

I would expect the follow button and the location, email, and urls to be displayed. Instead, when I inspect the sidebar, I see that the wrapper exists with all the right fields filled in, but the height of the container is set to zero for some reason.

## Steps to reproduce the behavior

I emptied the repo and made the Gemfile from scratch. I copied the _config.yml over and customized it. The profile wasn't appearing, so I added a `_pages` default. The information didn't appear, and it didn't appear on Github either.

## Other

Repo linked above.

**Comment by iBug** [comment]
The <kbd>Follow</kbd> button only shows on narrower screens (&lt; 1024px by default), and the author links are displayed in a pop-up after clicking that button.

![image](https://user-images.githubusercontent.com/7273074/108186076-ed225900-7147-11eb-8d07-16005720594e.png)

On wider screens (&ge; 1024px) the author links are laid out plain:

![image](https://user-images.githubusercontent.com/7273074/108186157-00352900-7148-11eb-8663-a0bd1cfbba89.png)

I'm not sure what you're expecting. Can you make it clearer what it's about?

**Comment by masasin** [comment]
I wasn't seeing anything at all.

![image](https://user-images.githubusercontent.com/4965180/108186622-e24ad280-710d-11eb-8a8a-370c40bd9efc.png)

I thought it was a configuration error, but it seems to be something wrong with uBlock Origin that I use. Turning it off made it appear, and also showed the social sharing in the footer. Thank you!

Now, I'm just curious why this happens, even when I put uBlock Origin into accept all mode.

**Comment by iBug** [comment, marked as answer]
You're probably using AdBlock Plus as well, which is [blocking social icons][1]. I recommend you get rid of ABP and stick to uBO - running two blockers isn't more beneficial than one.

  [1]: https://help.eyeo.com/adblockplus/block-social-media-icons-tracking

**Comment by masasin** [comment]
I wasn't. I was just using uBO. But I turned things on and off, and the ones that broke it were:

* AdGuard Social Media
* Fanboy's Annoyance
* Fanboy's Social

(All are under Annoyances.)
